From fbc1a34100f866b04b7f0fac3fac6fe578c5c585 Mon Sep 17 00:00:00 2001
From: Sanyog Kale <sanyog.r.kale@intel.com>
Date: Tue, 13 Dec 2016 11:17:20 +0530
Subject: [PATCH 0540/1132] ASoC: Intel: Kconfig and Makefile changes for
 SoundWire

This patch selects SoundWire Bus and SoundWire Master driver
when SND_SOC_INTEL_SKYLAKE config is selected.

Change-Id: I8f59f930a2c0089663e7976e354a72c78be8e03b
Signed-off-by: Sanyog Kale <sanyog.r.kale@intel.com>
Reviewed-on:
Reviewed-by: Diwakar, Praveen <praveen.diwakar@intel.com>
Reviewed-by: Koul, Vinod <vinod.koul@intel.com>
Reviewed-by: Nemallapudi, JaikrishnaX <jaikrishnax.nemallapudi@intel.com>
Reviewed-by: Singh, Guneshwor O <guneshwor.o.singh@intel.com>
Reviewed-by: Kp, Jeeja <jeeja.kp@intel.com>
Tested-by: Avati, Santosh Kumar <santosh.kumar.avati@intel.com>
---
 sound/soc/intel/Kconfig          | 2 ++
 sound/soc/intel/skylake/Makefile | 4 +---
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/sound/soc/intel/Kconfig b/sound/soc/intel/Kconfig
index 2f56dbf..487e444 100644
--- a/sound/soc/intel/Kconfig
+++ b/sound/soc/intel/Kconfig
@@ -113,6 +113,8 @@ config SND_SOC_INTEL_SKYLAKE
 	select SND_SOC_INTEL_SST
 	select SND_SOC_COMPRESS
 	select SND_SOC_ACPI_INTEL_MATCH
+	select SDW
+	select SDW_CNL
 	help
 	  If you have a Intel Skylake/Broxton/ApolloLake/KabyLake/
 	  GeminiLake or CannonLake platform with the DSP enabled in the BIOS
diff --git a/sound/soc/intel/skylake/Makefile b/sound/soc/intel/skylake/Makefile
index 7b8cf11..f36c0bb 100644
--- a/sound/soc/intel/skylake/Makefile
+++ b/sound/soc/intel/skylake/Makefile
@@ -13,9 +13,7 @@ snd-soc-skl-ipc-objs := skl-sst-ipc.o skl-sst-dsp.o cnl-sst-dsp.o \
 		skl-sst-cldma.o skl-sst.o bxt-sst.o cnl-sst.o \
 		skl-sst-utils.o skl-fwlog.o
 
-ifdef CONFIG_SDW
-  snd-soc-skl-ipc-objs += cnl-acpi.o
-endif
+ snd-soc-skl-ipc-objs += cnl-acpi.o
 
 obj-$(CONFIG_SND_SOC_INTEL_SKYLAKE) += snd-soc-skl-ipc.o
 
-- 
2.7.4

